src/sentry/runner/commands/backup.py

```diff
@@ -117,6 +137,12 @@ def import_():
     help="An optional comma-separated list of users to include. "
     "If this option is not set, all encountered users are imported.",
 )
+@click.option(
+    "--findings_file",
```

Comment:
```suggestion
    "--findings-file",
```

We don't have many commands that use `_` as the word separator. `-` is more common.

Reply:
Done. I've also aliased all of the existing flags to use dashes in addition to underscores. I kept the old naming to avoid breaking backwards compatibility for now; I'll deprecate it in the next release, then delete in the one after.